### ISSUE TITLE
CalDAV-Sync works only from client to server

### ISSUE THREAD

**Issue by tmst** [Kozea/Radicale#710]
At least with the 2.x release. It works fine (both ways) in the 1.1.6 release.

Existing tasks on the server are not seen by the client. Changes on the server are not synced on the client.

New tasks are synced to the server. Changes to existing tasks are synced to the server.

**Comment by Tntdruid** [comment]
Why use the old version then?

**Comment by Unrud** [comment]
Update Radicale to the latest version and start it with the ``--debug`` argument. Create a new calendar and add it in CalDAV-Sync. Then add a new event from another client and try to sync it with CalDAV-Sync. Post the full logging output here.

**Comment by vuntz** [comment]
@tmst just wondering: you mentioned tasks; does it work fine for events?

**Comment by tmst** [comment]
I observed this behavior in the 2.x branch which I installed using my system's package manager so I can start and stop it as a system service. Since the change away from using init.d/ on openSuse I don't know how to modify program arguments for system services. I have only tried this with tasks.

Just to be clear, I then uninstalled the 2.x release and installed the 1.x release and everything was hunky dory.
